### ISSUE TITLE
Fix for Dragonflight

### ISSUE THREAD

**Issue by spiralofhope** [spiralofhope/SuperDuperMacro#37]
TL;DR `twothumbs` researched a workaround:

This is a client bug.  For reference:

- https://github.com/Stanzilla/WoWUIBugs/issues/283
- https://wowpedia.fandom.com/wiki/CVar_ActionButtonUseKeyDown

The solution is to do:

```
/console ActionButtonUseKeyDown 0
```

As this problem hits many addons, it's likely Blizzard will solve this for release.

----

https://www.wowinterface.com/downloads/fileinfo.php?id=10496#cmnt491656

> ... LUA error when opening up the main macro window :

<details>
 <summary>Click to expand</summary>

```
Date: 2022-10-27 07:37:02ID: 17Error occured in: GlobalCount: 1Message: ...AddOns/SuperDuperMacro/SuperDuperMacro_Interface.lua line 1029:hooksecurefunc(): MacroFrame_Update is not a functionDebug:[string "=[C]"]: ?[string "=[C]"]: hooksecurefunc()[string "@Interface/AddOns/SuperDuperMacro/SuperDuperMacro_Interface.lua"]:1029: sdm_DefaultMacroFrameLoaded()[string "@Interface/AddOns/SuperDuperMacro/SuperDuperMacro_Core.lua"]:139:...face/AddOns/SuperDuperMacro/SuperDuperMacro_Core.lua:30[string "=[C]"]: LoadAddOn()[string "@Interface/FrameXML/UIParent.lua"]:530: UIParentLoadAddOn()[string "@Interface/FrameXML/UIParent.lua"]:609: MacroFrame_LoadUI()[string "@Interface/FrameXML/UIParent.lua"]:875:Interface/FrameXML/UIParent.lua:874[string "=[C]"]: ShowMacroFrame()[string "@Interface/FrameXML/ChatFrame.lua"]:2190: ?()[string "@Interface/FrameXML/ChatFrame.lua"]:5219:Interface/FrameXML/ChatFrame.lua:5165[string "=[C]"]: ChatEdit_ParseText()[string "@Interface/FrameXML/ChatFrame.lua"]:4883: ChatEdit_SendText()[string "@Interface/FrameXML/ChatFrame.lua"]:4919: ChatEdit_OnEnterPressed()[string "*ChatFrame.xml:127_OnEnterPressed"]:1:[string "*ChatFrame.xml:127_OnEnterPressed"]:1
```
</details>

With no investigation, I think that commenting-out 1029 to 1061 in SuperDuperMacro_Interface.lua might solve this.

The problem would be that /macro would let them tinker and break SDM macros, so.. don't do that.  :)

This actually seems kinda straightforward to fix, but I forgot Lua and I'm not playing any more.
